hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerChecksumTreeManager.java

```diff
@@ -157,6 +173,14 @@ private void write(KeyValueContainerData data, ContainerProtos.ContainerChecksum
     }
   }
 
+  public File getContainerChecksumFile(KeyValueContainerData data) {
+    return new File(data.getMetadataPath(), data.getContainerID() + ".tree");
+  }
+
+  public ContainerMerkleTreeMetrics getMetrics() {
```

Comment:
```suggestion
  @VisibleForTesting
  public ContainerMerkleTreeMetrics getMetrics() {
```
This is only needed for test code right?

Reply:
Right now we are using it for just test. In future we may use it so kept it as it is, but I'll add `@VisibleForTesting`. We can remove it when needed.